src/containers/audio-rewards-page/components/modals/VerifiedUpload.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Button,
+  ButtonType,
+  IconInstagram,
+  IconTwitterBird,
+  IconUpload
+} from '@audius/stems'
+import TwitterButton from 'components/general/TwitterButton'
+import React, { useCallback } from 'react'
+import { useModalState } from 'store/application/ui/modals/hooks'
+import ModalDrawer from './ModalDrawer'
+import InstagramButton from 'components/general/InstagramButton'
+
+import styles from './VerifiedUpload.module.css'
+import { useNavigateToPage } from 'hooks/useNavigateToPage'
+import { UPLOAD_PAGE } from 'utils/route'
+import { useWithMobileStyle } from 'hooks/useWithMobileStyle'
+
+const messages = {
+  title: 'Verified Upload',
+  step1Title: 'Step 1: ✅ Link Verified Social Media Accounts',
+  step1Subtitle: 'Link your verified Twitter or Instagram Account',
+  verifyTwitterButton: 'Verify With Twitter',
+  verifyIGButton: 'Verify With Instagram',
+  step2Title: 'Step 2: Upload a Track',
+  step2Subtitle: 'Upload some music to your verified Account',
+  uploadButton: 'Upload',
+  step3Title: 'Step 3: Tag us And Let Us Know',
+  step3Subtitle:
+    'Post a link to your track from your verified Twittter or Instagram and tag us',
+  findUsTwitter: 'Find Us On Twitter',
+  findUsInstagram: 'Find Us On Instagram'
+}
+
+const TWITTER_LINK = 'https://twitter.com/AudiusProject'
+const IG_LINK = 'https://www.instagram.com/audiusmusic/?hl=en'
+
+const onClickTwitter = () => {
+  window.open(TWITTER_LINK, '_blank')
+}
+
+const onClickInstagram = () => {
+  window.open(IG_LINK, '_blank')
+}
+
+const Divider = () => <div className={styles.divider} />
+
+const VerifiedUpload = ({ dismissModal }: { dismissModal: () => void }) => {
+  const navigate = useNavigateToPage()
+
+  const onClickUpload = useCallback(() => {
+    navigate(UPLOAD_PAGE)
+    dismissModal()
+  }, [navigate, dismissModal])
+
+  const wm = useWithMobileStyle(styles.mobile)
+  return (
+    <div className={wm(styles.container)}>
+      <span className={styles.title}>{messages.step1Title}</span>
+      <span className={styles.subtitle}>{messages.step1Subtitle}</span>
+      <div className={styles.verifyButtons}>
+        <TwitterButton
+          textLabel={messages.verifyTwitterButton}
+          size='large'
+          className={styles.twitterButton}
+        />
+        <InstagramButton
+          // TODO: handle these
```

Comment:
Is this intentionally left not finished to be done later?

Reply:
yep. sry was unclear